src/cargo.rs

```diff
@@ -54,3 +54,17 @@ pub fn run(cmd: &str) {
     Err(e) => println!("Failed to execute 'cargo {}': {}", cmd, e)
   };
 }
+
+/// Runs cargo by passing through a set of multiple commands and displays the output.
+pub fn run_raw(cmds: &str) {
```

Comment:
Given above comment above obsoleting the `--<action>` options, `run_raw` could simply replace `run`. Simply add a check (in compile.rs) that if `cmd` is empty or only whitespace, replace it with `build test`.